meeting-minutes/2023/2023-05-04-TOC-meeting-record.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: 2023-05-04 TOC Meeting Record
+parent: 2023
+grand_parent: Meeting Minutes
+---
+![Antitrust Policy Notice](../images/antitrust-policy-notice.png "Antitrust Policy Notice")
+![All are Welcome in the Hyperledger Community](../images/all-are-welcome.png "All are Welcome in the Hyperledger Community")
+
+Hyperledger is committed to creating a safe and welcoming community for all. For more information please visit our Code of Conduct: [Hyperledger Code of Conduct](https://toc.hyperledger.org/governing-documents/code-of-conduct.html)
+
+# Announcements
+* The [Hyperledger /dev/weekly developer newsletter](https://wiki.hyperledger.org/pages/viewpage.action?pageId=39618905) goes out each Friday to hundreds of Hyperledger developers. It is a collaborative effort. If you have a project release, pull request, community event, and/or relevant article you would like highlighted next week, please [leave a comment for consideration on the upcoming newsletter wiki page](https://wiki.hyperledger.org/display/DR/2023)
+* Review [Security Policy Vulnerability Disclosure Template](https://docs.google.com/document/d/1ne2RISsvRpOzy4buL1WTvs3Wm1_Scc6_82hKHx60Chk/edit).
+
+# Quarterly reports
+* Please review any [outstanding quarterly reports](https://github.com/hyperledger/toc/pulls?q=is%3Apr+is%3Aopen+label%3Aquarterly-report+user-review-requested%3A%40me).
+
+# Past due reports
+* 2023 Q1 Hyperledger Transact (**due 23 Feb 2023**)
+    * [Reminder to maintainers sent March 2](https://discord.com/channels/905194001349627914/941414458922790982/1080893688441491536)
+    * [Ongoing discussion with maintainers](https://discord.com/channels/905194001349627914/941414458922790982/1081247988275486752)
+    * [Waiting on list of maintainers from Shawn](https://discord.com/channels/905194001349627914/941414458922790982/1083097678461800580) to determine if they want to move project to Dormant
+    * Peter volunteered to open [issue](https://github.com/hyperledger/transact/issues/375) and [pull request](https://github.com/hyperledger/transact/pull/376) stating that the work is being moved to libsawtooth (opened on April 4, 2023)
+    * No update on issue or pull request as of April 11, 2023
+    * Arun reached out directly to Andrea and Shawn regarding the pull request, and it has since been merged
+    * Daniela will reach out to Shawn about status and moving to EOL
+
+# Upcoming reports
+* 2023 Q2 Hyperledger Sawtooth (due 27 Apr 2023)
```

Comment:
```suggestion
```
This is now overdue. Please send a reminder to the Sawtooth community and link above.

Reply:
Done, shared the latest update.